tsconfig.json

```diff
@@ -7,12 +7,13 @@
       "es6",
       "dom"
     ],
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
Wouldn't "es6" be sufficient?

Reply:
I wonder if the compiler options in `karma.conf.js` have to be adapted accordingly.